CMakeLists.txt

```diff
@@ -238,3 +251,40 @@ if(CLANG_FORMAT)
         COMMAND "${CLANG_FORMAT}" -i -verbose ${TLS12_DOWNLOAD_SOURCES}
     )
 endif()
+
+# ====================
+# === Installation ===
+# ====================
+
+include(CMakePackageConfigHelpers)
+
+install(TARGETS vcpkg
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+    BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+
+install(TARGETS vcpkglib
+    EXPORT vcpkgTargets
+    RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    BUNDLE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
+
+install(EXPORT vcpkgTargets
+    FILE vcpkgConfig.cmake
+    NAMESPACE vcpkg::
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/vcpkg"
+)
```

Comment:
Since we're not shipping libraries, I don't think we need a config file.

Reply:
Also, this is just random but if we do need a config file, we should be using `vcpkg-targets.cmake` and `vcpkg-config.cmake`. Also, probably want to do the standard `configure_package_config_file`.